### DISCUSSION TITLE
Remove items from Shell Menu

### DISCUSSION THREAD

**Discussion by marcdsdavid** [wezterm/wezterm#3670] [Q&A] [answered]
Hello! I am quite new to WezTerm and still in the process of configuring it I was looking to remove some items from the Shell Menu

this one: 
<img width="447" alt="CleanShot 2023-05-03 at 12 04 18@2x" src="https://user-images.githubusercontent.com/7226788/235898960-0625514c-66de-4392-b609-8b77da260a8a.png">

I can't seem to find a way of doing this, I see that I can add items using launch_menu but I would like to remove some of the default ones i.e "New Tab" as I don't intend to use it and have removed the shortcut for it.

**Comment by wez** [comment, marked as answer]
You cannot remove entries from the menu bar, only remove their default key assignments.
This is intentional: the primary purpose of the menu is discoverability and accessibility
of the various built in functions.
